### PR TITLE
feat: add PoC of v2 for languages with attributes

### PR DESCRIPTION
**Pull request by gabrielfs7**
https://oat-sa.atlassian.net/browse/AUT-2332

# Solution

- Accept the header `Accept-version` to version the languages api. To get new contract, please provide `v2` in the header.
- Add a richer object to represent the languages, including orientation.

# TODO

- [x] Migrate PoC code to a proper implementation.
- [x] Add unit tests

### CODE REVIEW

File: migrations/Version202207041535182167_tao.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public function getDescription(): string
+    {
+        return 'Update Ontology models';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addReport(
+            $this->propagate(new SyncModels())([])
+        );
```
Comment (shpran) [inline migrations/Version202207041535182167_tao.php:27]: You can use `$this->runAction()` instead 🙂

File: migrations/Version202207041535182167_tao.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+use oat\tao\scripts\SyncModels;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202207041535182167_tao extends AbstractMigration
+{
+
+    public function getDescription(): string
```
Comment (shpran) [inline migrations/Version202207041535182167_tao.php:18]:
```suggestion
{
    public function getDescription(): string
```

File: migrations/Version202207041535182167_tao.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\tao\scripts\SyncModels;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202207041535182167_tao extends AbstractMigration
```
Comment (shpran) [inline migrations/Version202207041535182167_tao.php:15]:
```suggestion
final class Version202207041535182167_tao extends AbstractMigration
```

**Review by shpran — APPROVED** [review]
Great job! 👍🏻 

- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: migrations/Version202207041535182167_tao.php (hunk 2 above)
Comment (gabrielfs7) [reply migrations/Version202207041535182167_tao.php]: I did not get this change request. Can you please explain?

**Review by hectoras — APPROVED** [review]
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Target branch is correct
- [x] Commits are following conventional commits
- [x] New code is covered by tests (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Kitchen links are not exposed in the description

File: migrations/Version202207041535182167_tao.php (hunk 2 above)
Comment (meefox) [reply migrations/Version202207041535182167_tao.php]: It looks like removing an extra new line between

**Comment by meefox** [comment]
Screenshot 2022-07-06 at 18 15 42
is there any benefits to have ontology instead of rtl/ltr values?

**Review by meefox — COMMENTED** [review]
Screenshot 2022-07-06 at 18 15 42
can we send just RTL/LTR values instead, or there is advantage of sending it like this?
I am not sure that we are going to use URI at this point. 
But it can make the size of the requests smaller.

File: views/package.json [hunk 4]
```diff
@@ -16,8 +16,8 @@
         "@oat-sa/expr-eval": "1.3.0",
         "@oat-sa/tao-core-libs": "0.4.5",
         "@oat-sa/tao-core-sdk": "1.19.2",
-        "@oat-sa/tao-core-shared-libs": "1.2.0",
```
Comment (meefox) [inline views/package.json:19]: - [ ] Reminder to update released versions before merge, or revert to original versions

**Review by meefox — APPROVED** [review]

**Pull request merged**